code_challenges.rb

```diff
@@ -31,3 +54,18 @@
 # Expected output: 'LEARN is not a palindrome'
 is_palindrome3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+def palindrome_vibe_check (str)
+  if str.downcase.reverse == str.downcase
+    p "#{str} is a palindrome and passes the vibe check."
+  else
+    p "#{str} does not pass the vibe check."
+  end
+end
+
```

Comment:
Great work on all three of these! This material is kind of elementary so I understand not using pseudo code. If you do find something interesting as you go along feel free to make note of it!